app/logic/Calendar/Event.ts

```diff
@@ -240,7 +240,15 @@ export class Event extends Observable {
     );
   }
 
+  /**
+   * Starts editing an event, if it hasn't already started being edited.
+   * This can happen when switching between the event editor and another
+   * screen, or certain updates within editing events can also trigger it.
```

Comment:
```suggestion
   * Starts editing an event.
   *
   * If it is already being edited, this is a no-op. This can happen when switching
   * between the event editor and another screen, or certain updates within
   * editing events can also trigger it.
```